src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -137,3 +137,69 @@
         - name: --ids
           short-summary: "One or more scale set or specific VM instance IDs. If provided, no other 'Resource Id' arguments should be specified."
 """
+
+helps['vm get-instance-view'] = """
+    type: command
+    short-summary: "Get's a vm including instance information (powerState)"
+"""
+
+helps['vm'] = """
+    type: group
+    short-summary: Provision Linux and Windows virtual machines in minutes
+"""
+helps['vm access'] = """
+    type: group
+    short-summary: Manage user access
+"""
+helps['vm availability-set'] = """
+    type: group
+    short-summary: Group resources into availability-sets for high-availability requirements
+"""
+helps['vm boot-diagnostics'] = """
+    type: group
+    short-summary: Troubleshoot virtual machine start-up
+"""
+helps['vm container'] = """
+    type: group
+    short-summary: Use Docker-based tools to deploy and manage containers
+"""
+helps['vm diagnostics'] = """
+    type: group
+    short-summary: Configure the Azure VM diagnostics extension
+"""
+helps['vm disk'] = """
+    type: group
+    short-summary: Manage vm storage disks
+"""
+helps['vm extension'] = """
+    type: group
+    short-summary: Extend the functionality of your VMs with vm extensions
+"""
+helps['vm extension image'] = """
+    type: group
+    short-summary: Find vm extensions available for your subscription and region
```

Comment:
Is the casing of vM supposed to be consistent - right now, it is mixed between VM and vm in quite a few locations. And are we supposed to refer to it as a Virtual Machine or VM?